### DISCUSSION TITLE
Steam Deck Decky Plugin issue

### DISCUSSION THREAD

**Discussion by nirvana-phoenix** [open-goal/jak-project#3645] [Q&A]
![IMG_4849](https://github.com/user-attachments/assets/716b7e3c-6172-4a2e-8a0f-96cd8f3af9fe)

Apologies if this is in the wrong place. The Decky Plugin is saying the games are out of date and I'm not sure how to update them.

I downloaded the launcher thinking that would work but it doesn't recognise both games are installed (it wants iso files, and they've already been converted to the PC versions).

How can I update these games on Deck?

Thanks very much in advance!

**Comment by xTVaser** [comment]
If those buttons are disabled it means one of two things:
- it can't find your jak 1 ISO (it's required to update it, sounds like that's probably your option)
- for some reason the game's install status was not saved properly.  In this case, you should delete and reinstall (the game via the Delete button, not the plugin).

The decky plugin is an interim replacement for the launcher so that's completely separate and not relevant if you are using the decky plugin.
